apps/frontend/src/network/users/Outputs.tsx

```diff
@@ -60,7 +63,10 @@ const OutputsList: React.FC<OutputsListProps> = ({
     algoliaResultsToStream(
       createCsvFileStream(
         { headers: true },
-        `SharedOutputs_${format(new Date(), 'MMddyy')}.csv`,
+        `SharedOutputs_${firstName}${lastName}_${format(
```

Comment:
Why you didn't apply titleCase in this case?
`firstName` and/or `lastName` could be in lowercase or uppercase